ex2.py

```diff
@@ -1,5 +1,4 @@
 #reversed user's first and last name
-
 def rev(x): #the required function, x represents a string 
     st = x.split(" ") #splitting the string (x) into list (data items)
```

Comment:
only missing the fact that the string is divided according to the " " character given as its input. 
a question for you: what happens if your string has not character as " "